Makefile

```diff
@@ -5,90 +5,77 @@ SHELL := bash
 HIDE:=$(if $(VERBOSE),,@)
 UNAME := $(shell uname)
 
-.PHONY: kat rebuild update clean migrate build itest debian-build-image ubuntu-build-image
+.PHONY: $(MAKECMDGOALS)
 
 # Export Docker buildkit options
 export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
 
-kat: env-if-empty clean # This should give you a clean install
-	make build
-	make up
+# Build and bring up all containers (default target)
+kat: env-if-empty build up
 
-rebuild: clean
-	make build
-	make up
+# Update using git pull and bring up containers
+update: pull kat
 
-update: down pull
-	make build
-	make up
-
-clean: down # This should clean up all persistent data
-	-docker volume rm nl-kat-coordination_postgres-data nl-kat-coordination_bytes-data nl-kat-coordination_xtdb-data
-
-export version
-
-upgrade: fetch down # Upgrade to the latest release without losing persistent data. Usage: `make upgrade version=v1.5.0` (version is optional)
-ifeq ($(version),)
-	version=$(shell curl --silent  "https://api.github.com/repos/minvws/nl-kat-coordination/tags" | jq -r '.[].name' | grep -v "rc" | head -n 1)
-	make upgrade version=$$version
-else
-	make checkout branch=$(version)
-	make build-all
-	make up
-endif
-
-reset:
-	-docker-compose down --remove-orphans --volumes --timeout 0
-	make up
-	make -C boefjes build
-	make -C rocky almost-flush
+# Bring down containers, remove all volumes, and bring them up again
+reset: clean kat
```

Comment:
I suppose there's no need for the almost-flush functionality anymore?

Reply:
I removed it because it can be invoked using the Rocky Makefile directly. If needed we could make a target that invokes it in the main Makefile?

Reply:
Yes perhaps, although I am not sure it was really being used (which we might find out by removing it altogether of course 😀)

Reply:
Let's keep it out for now and add it back if anyone requests it.